docs/extend/plugins.md

```diff
@@ -68,6 +69,9 @@ Plugin
 [Virtuozzo Storage and Ploop plugin](https://github.com/virtuozzo/docker-volume-ploop) | A volume plugin with support for Virtuozzo Storage distributed cloud file system as well as ploop devices.
 [VMware vSphere Storage Plugin](https://github.com/vmware/docker-volume-vsphere)    | Docker Volume Driver for vSphere enables customers to address persistent storage requirements for Docker containers in vSphere environments. 
 
+[azure-fs]: https://azure.microsoft.com/blog/azure-file-storage-now-generally-available/
+[azurefile-plugin]: https://azure.microsoft.com/blog/persistent-docker-volumes-with-azure-file-storage/
```

Comment:
Oh, can you change these to inline links, instead of references? I know this works, but we don't really use them anywhere, so for consistency 😇